barberia/src/App/pages/TimePicker.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import AppointmentPicker from "appointment-picker";
+import "../../../node_modules/appointment-picker/dist/appointment-picker.css";
```

Comment:
it's better to just do  `import "appointment-picker/dist/appointment-picker.css"`, since in case the file location actually changed then this will error